packages/server-admin-ui/src/views/ServerConfig/Logging.js

```diff
@@ -70,8 +70,11 @@ class Settings extends Component {
               </CardBody>
               <CardFooter>
                 <small className='text-muted'>
-                  Click button to download each logfile
+                  Click button to download each logfile or
                 </small>
+                <a href={`${window.serverRoutesPrefix}/ziplogs`}>
+                  <Button className='m-2'>Get all logs in one ZIP</Button>
```

Comment:
I would change the end to read _in one ZIP file_

Reply:
Done